### PR TITLE
Add copy button to encoded + decoded fields

### LINKED ISSUE #818: [FEATURE] Dashboard | Add copy button to encoded fields
### Detailed description

Go to the dashboard/encoder  : 

image
image
image

### Context

I think that we should add copy button next to every line so we can copy the encoded data easily 

### Possible implementation

_No response_

### Additional information

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Tamir198**
Fixed #818;

Now we have a button to copy the encoded and decoded texts : 

https://github.com/thecyberworld/TheCyberHUB/assets/34707669/438a2608-4956-4be5-a867-a55b095e45b6

Please notice that I pushed the package lock because the dependencies of font awesome was needed but was missing from the file.

### CODE REVIEW

**Review by kabir0x23 — APPROVED** [review]
awesome 

thanks for raising a pr

File: package-lock.json
Comment (kabir0x23) [inline package-lock.json:1]: what are the changes with package-lock file?

Comment (Tamir198) [reply package-lock.json:1]: @kabir0x23  
In another (already merged) pr, there was addition of font awsome but the person who added the package didn`t pushed the package lock. 
So now if you will pull the dev branch you will get an error that you are missing this package, thus the package lock changed are needed.

Comment (kabir0x23) [reply package-lock.json:1]: okay okay, understood

**Pull request merged**